powerauth-test-server/deploy/liquibase/readme.txt

```diff
@@ -0,0 +1,22 @@
+================================================================================
+                          PowerAuth Test Server
+                   Liquibase Setup Instructions
+================================================================================
+
+Directory Purpose:
+------------------
+The directory is dedicated to storing Liquibase scripts for Docker build context.
+
+Setup Instructions:
+-------------------
+A) Create folder 'data' in powerauth-test-server/deploy/liquibase'.
```

Comment:
Very verbose. But ok. Only one point you do not need to create data directory. Either run copy command or execute supplied script. That's all you need to do.